scripts/release.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+ember build --production && npm publish
```

Comment:
`--production` is not a valid option. It should be `ember build --environment production` or `ember build -prod`.